source/_drafts/Update-on-the-state-of-Tox.md

```diff
@@ -0,0 +1,87 @@
+title: Update on the state of Tox
+date: 2015-02-18 13:49:11
+author: Dubslow
+tags:
+---
+
+Hello everyone. We realize it's been a while since the last post, but a lot has
+happened and Tox continues to progress, so there's a lot to talk about.
+
+#Tox core
+
+[toxcore](https://github.com/irungentoo/toxcore) has been largely in maintenance mode,
+with memory leaks, logic errors and other bugs being fixed every day. Recently
+work has begun on implementing a [new, cleaner API](https://github.com/irungentoo/toxcore/tree/new_api)
+written by the [Tox4j](https://github.com/sonOfRa/tox4j) developers.
+
+Additionally, work is progressing on the complete [groupchat re-design](https://github.com/JFreegman/toxcore)
+and overhaul, started by [alnf](https://github.com/alnf) with currently being
+lead primarily by [JFreegman](https://github.com/JFreegman). Features include moderation
+abilities and group chat persistence (!). These group chats are currently a long
+way from being ready to merge into master; however, there is currently a properly
+[modified build of Toxic](https://jenkins.libtoxcore.so/job/toxic_linux_beta/lastSuccessfulBuild/artifact/toxic)
+for GNU-Linux available with the changes. Note that important packet IDs have been changed
+from the primary Tox network for testing purposes, so don't use that binary as your regular
+client. Please also continually update it if you use it, because breaking changes
+to the new groupchats are continually being made as they undergo primary
+development, and old nodes on the testing network can make testing difficult for
+updated nodes with newer code. You can follow development on the #tox-groupchats
+IRC channel on Freenode.
+
+#Audio filtering and echo cancellation
+
+It's been around for a few months now, but [uTox](https://github.com/notsecure/uTox)
+has had audio filtering, provided by [code](https://github.com/irungentoo/filter_audio) from [WebRTC](https://code.google.com/p/webrtc/).
+([qTox](https://github.com/tux3/qTox) theoretically has had the same code, but due
+to a bug, did not. That was bug was fixed yesterday.)
```

Comment:
Instead of saying that it was supposed to work, but it didn't, but now it actually does -- maybe just say that qTox supports it too?